{"name": "test_admin_login[case3_用户名密码错误的登录]", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <case.test_member_controller.test_login.Test_DengLu object at 0x000001D3706E1EF0>\ncase_data = {'body': {'password': '9999997', 'username': 'adminhhhhhhhhhh'}, 'case_name': 'case3_用户名密码错误的登录', 'expect': {'code': 500, 'message': 'Bad credentials'}}\nmember_controller_z = <api.admin.member_controller_api.MemberControllerApi object at 0x000001D37063FEF0>\n\n    @decorate_log\n    @pytest.mark.parametrize(\"case_data\", data,ids=[data[i]['case_name'] for i in range(len(data))])  # 参数化测试用例\n    def test_admin_login(self,case_data,member_controller_z):\n        time.sleep(5)\n        print(\"case_data\",case_data)\n        case_body = case_data['body']\n        \"\"\"请求登录测试用例\"\"\"\n        with allure.step('登录步骤'):\n            actual_data = member_controller_z.login(password=case_body['password']\n                           ,username=case_body['username'])\n            \"\"\"断言\"\"\"\n            print(\"566666:\",actual_data)\n>           assert_api(actual_data,case_data['expect'])\n\ntest_login.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\common\\logger.py:68: in log\n    raise e\n..\\..\\common\\logger.py:64: in log\n    func(*args, **kwargs)\n..\\..\\common\\assert_api.py:36: in assert_api\n    raise e\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nactual_data = {'code': 200, 'data': {'expiresIn': 86399, 'refreshToken': 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJhZ...yTn6lZ62jo9O-Luq1qIWwExF6dJGOM8DcsYmkDnIwqMT8DhXeQynaNh8UR9fHNA0Ur2ZwEjyI', 'tokenHead': 'Bearer '}, 'message': '操作成功'}\nexpect_data = {'code': 500, 'message': 'Bad credentials'}\n\n    @decorate_log\n    def assert_api(actual_data,expect_data):\n        # Logger.info(\"*************** {}开始执行用例 ***************\".format(expect_data['info']))\n        Logger.info(\"\\r\\nassert:\\r\\n 实际结果：{}，\\r\\n期望结果{}\".format(actual_data, expect_data))\n        try:\n            a = expect_data\n            b = actual_data\n            for i, j in a.items():\n                if i in b.keys():\n>                   assert j == b[i]\nE                   AssertionError\n\n..\\..\\common\\assert_api.py:27: AssertionError"}, "steps": [{"name": "登录步骤", "status": "failed", "statusDetails": {"message": "AssertionError\n", "trace": "  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\case\\test_member_controller\\test_login.py\", line 37, in test_admin_login\n    assert_api(actual_data,case_data['expect'])\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\logger.py\", line 68, in log\n    raise e\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\logger.py\", line 64, in log\n    func(*args, **kwargs)\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\assert_api.py\", line 36, in assert_api\n    raise e\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation2\\common\\assert_api.py\", line 27, in assert_api\n    assert j == b[i]\n"}, "steps": [{"name": "响应日志", "status": "passed", "parameters": [{"name": "status_code", "value": "200"}, {"name": "text", "value": "{'code': 200, 'message': '操作成功', 'data': {'token': 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJhZG1pbiIsInNjb3BlIjpbImFsbCJdLCJpZCI6MjQzNDQyNywiZXhwIjoxNjQ4MTk3NzM0LCJhdXRob3JpdGllcyI6WyLliY3lj7DkvJrlkZgiXSwianRpIjoiMjQzYmRiMDAtOTlhMi00Y2ZjLTlkMTQtN2RjODhmNWMyZDI4IiwiY2xpZW50X2lkIjoicG9ydGFsLWFwcCJ9.SkVlybCSO9sxqqRaFbmEdrb-zWT7NnNMlktXcQGty6dBUA2_7WEI4tdzlF8_KRyEzicS-M-iZG6i-oXiosFgrPerR4CtI1yjCjyTn6lZ62jo9O-Luq1qIWwExF6dJGOM8DcsYmkDnIwqMT8DhXeQynaNh8UR9fHNA0Ur2ZwEjyI', 'refreshToken': 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX25hbWUiOiJhZG1pbiIsInNjb3BlIjpbImFsbCJdLCJhdGkiOiIyNDNiZGIwMC05OWEyLTRjZmMtOWQxNC03ZGM4OGY1YzJkMjgiLCJpZCI6MjQzNDQyNywiZXhwIjoxNjQ4NzE2MTM0LCJhdXRob3JpdGllcyI6WyLliY3lj7DkvJrlkZgiXSwianRpIjoiY2I0ZmU1ZmItY2EwMS00Mjk2LWE5YTEtZWE3MGI5YWY2M2FlIiwiY2xpZW50X2lkIjoicG9ydGFsLWFwcCJ9.Xq9u1SXCN8gXEzxbBYr0lUkkHxnRCMDp_jgD_2KPnEhfM9oLkFbc_MWk8TncXuTXKJaFl-Ss_-cvfKEYEl0m8UZNSFxDw08UaLZ9or4ruRpDB8sELO0SO1q0V3wIbZqdhOI6CRMVeNXXSz2OvefwzGe5xM4DtEk3OFY5SVv9sXw', 'tokenHead': 'Bearer ', 'expiresIn': 86399}}"}], "start": 1648111333960, "stop": 1648111333960}, {"name": "请求日志", "status": "passed", "parameters": [{"name": "url", "value": "'http://121.37.169.128:8201/mall-member/sso/login?platform=test&time=1648111333358&username=admin&password=123456'"}, {"name": "method", "value": "'post'"}, {"name": "body", "value": "{}"}, {"name": "headers", "value": "{'Content-Type': 'application/json'}"}], "start": 1648111333960, "stop": 1648111333960}], "start": 1648111333358, "stop": 1648111333961}], "attachments": [{"name": "log", "source": "6ed4bf4f-4b25-4d1a-8127-6b62ccd1fb93-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case_data", "value": "{'case_name': 'case3_用户名密码错误的登录', 'body': {'username': 'adminhhhhhhhhhh', 'password': '9999997'}, 'expect': {'message': 'Bad credentials', 'code': 500}}"}], "start": 1648111328347, "stop": 1648111333962, "uuid": "08371765-ff6b-4b00-92dc-8c677feb05a8", "historyId": "e8e60fbd9f64690605ffb6f55812c06c", "testCaseId": "02c7f242c9f09fbdefa4b8f5b820915e", "fullName": "case.test_member_controller.test_login.Test_DengLu#test_admin_login", "labels": [{"name": "story", "value": "登录功能"}, {"name": "feature", "value": "测试member_controller模块"}, {"name": "parentSuite", "value": "case.test_member_controller"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_DengLu"}, {"name": "host", "value": "LAPTOP-9964GMTD"}, {"name": "thread", "value": "52556-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case.test_member_controller.test_login"}]}